src/pages/Embeds/Plio.vue

```diff
@@ -191,6 +191,7 @@ export default {
       isScorecardShown: false, // to show the scorecard or not
       plioTitle: "", // title of the plio
       isAspectRatioChecked: false, // whether the check for aspect ratio has been done
+      watchingEventDBId: null, // the DB id of the latest 'watching' event for a given session
```

Comment:
I think there is no other id it can be apart from the DB id - plus you've mentioned it already in the comments.
```suggestion
      watchingEventId: null, // the DB id of the latest 'watching' event for a given session
```